docs/the-new-architecture/pillars-turbomodule.md

```diff
@@ -780,7 +780,7 @@ import {
   Text,
   Button,
 } from 'react-native';
-import RTNCalculator from 'rtn-calculator/js/NativeCalculator.js';
```

Comment:
Can you also update the same page on the 0.71 docs?

Reply:
And the 0.70 as well :)

Reply:
Done 👍